ibis/tests/expr/test_sql_builtins.py

```diff
@@ -148,7 +148,7 @@ def test_sign(functional_alltypes, lineitem):
 def test_round(functional_alltypes, lineitem):
     result = functional_alltypes.double_col.round()
     assert isinstance(result, ir.IntegerColumn)
-    assert result.op().args[1] is None
+    assert result.op().args[1] == ibis.literal(0).op()
```

Comment:
this seems like it is testing the implementation and I would just remove it, but also fine with keeping it.

Reply:
Going to keep it for now.